content/news/buzz/september-2024-status-update/contents.lr

```diff
@@ -0,0 +1,63 @@
+
+* We released `Toga 0.4.7 <https://pypi.org/project/toga/0.4.7/>`__, incorporating fixes for a number of bugs discovered in the recent 0.4.6 release, as well as some minor feature additions.
+
+* We transitioned Toga's GTK backend to use `PyGObject's native asyncio support <https://github.com/beeware/toga/pull/2550>`__. This allowed us to deprecate `Gbulb <https://github.com/beeware/gbulb>`__, the GTK asyncio plugin that we've been maintaining for several years.
+
+* We corrected an issue with the `use of recent Textual versions in Toga <https://github.com/beeware/toga/pull/2822>`__, and `added a bare-bones runtime test <https://github.com/beeware/toga/pull/2823>`__ to ensure that the problem doesn't re-occur. This also required a fix for `macOS console app testing in Briefcase <https://github.com/beeware/briefcase/pull/1992>`__.
+
+* We made some `usability improvements to the rendering of Toga's documentation <https://github.com/beeware/toga/pull/2831>`__, and modified Toga's ReadTheDocs configuration to `encourage search engines to return the stable documentation by default <https://github.com/beeware/toga/pull/2833>`__.
+
+* We modified the handling of Toga's app icons on macOS to `improve compatibility with PyInstaller <https://github.com/beeware/toga/pull/2852>`__.
+
+* We corrected a number of issues with WebView handling, including `ensuring that DOM storage is always available <https://github.com/beeware/toga/pull/2815>`__, and `improving error handling when the WebView cannot be initialised on Windows <https://github.com/beeware/toga/pull/2781>`__.
+
+* We added protection against an edge case of asynchronous Task handling that could lead to the `premature garbage collection of long lived tasks <https://github.com/beeware/toga/pull/2814>`__. We've also had some discussions with the CPython core team about `fixing this problem in Python itself <https://github.com/python/cpython/pull/121264>`__.
+
+* We spoke at `PyBay <https://pybay.org>`__ on building mobile apps in Python.
+
+What's next?
+--------------
+
+In October, we will continuing working on binary package support for mobile platforms. In addition to responding to any review feedback on the patches we've already submitted, we'll be working on new patches for ``cibuildwheel`` and ``meson-python``, as well as tooling to make it easy for third-party libraries to run CPython test suites on mobile platforms. We should also be able finalise Briefcase's Python 3.13 support for Android, and start work on Android packaging. Supporting packaging for mobile platforms is a long term project which we expect to continue for several months; the hope is that by the end of the year, we will be able to contribute patches to third-party projects so that iOS and Android wheels become officially produced, rather than BeeWare maintained artefacts. 
```

Comment:
```suggestion
In October, we will continuing working on binary package support for mobile platforms. In addition to responding to any review feedback on the patches we've already submitted, we'll be working on new patches for ``cibuildwheel`` and ``meson-python``, as well as tooling to make it easy for third-party libraries to run CPython test suites on mobile platforms. We should also be able finalise Briefcase's Python 3.13 support for Android, and start work on Android packaging. Supporting packaging for mobile platforms is a long term project which we expect to continue for several months; the hope is that by the end of the year, we will be able to contribute patches to third-party projects so that iOS and Android wheels become officially produced, rather than BeeWare-maintained artefacts. 
```